com.tiny-beast-dictionary.yml

```diff
@@ -0,0 +1,27 @@
+app-id: org.Tiny-beast-dictionary
```

Comment:
this app-id is INVALID. It must have at least 3 components, the first two being derivated from a domain under the developer control.

https://docs.flatpak.org/en/latest/conventions.html#application-ids